### DISCUSSION TITLE
No pinecone support?

### DISCUSSION THREAD

**Discussion by giachat** [langflow-ai/langflow#251] [General]
hey, cool project guys! Under vector store I am only seeing "Chroma". Does lanflow support pinecone vector store?

Thanks

**Comment by ogabrielluiz** [comment]
Hey @giachat , we haven't added any other vector stores yet because we need to test them and we haven't got around to that yet.

**Reply by Elliot-Construct** [reply]
I'm versed in Weaviate. How can I help?

**Reply by ogabrielluiz** [reply]
Hi, Elliot! 

I've opened issue #291 to track the supported vector stores. If you want to discuss specifics just leave a comment there or try your hand in the dev branch.
